<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1920</width>
    <height>1080</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1920</width>
    <height>1080</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Only Window</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="minimumSize">
    <size>
     <width>1920</width>
     <height>1080</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>1920</width>
     <height>1080</height>
    </size>
   </property>
   <widget class="QTabWidget" name="tabWidget">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>1900</width>
      <height>1060</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="tabAlpha">
     <attribute name="title">
      <string>Tab α</string>
     </attribute>
     <widget class="QWidget" name="verticalLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>-1</x>
        <y>-1</y>
        <width>1891</width>
        <height>1031</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="tabAlphaVerticalLayout">
       <item>
        <widget class="QLCDNumber" name="lcdNumber">
         <property name="smallDecimalPoint">
          <bool>true</bool>
         </property>
         <property name="segmentStyle">
          <enum>QLCDNumber::Filled</enum>
         </property>
         <property name="value" stdset="0">
          <double>0.000000000000000</double>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="horizontalSlider">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>250</height>
          </size>
         </property>
         <property name="cursor">
          <cursorShape>ArrowCursor</cursorShape>
         </property>
         <property name="minimum">
          <number>-500</number>
         </property>
         <property name="maximum">
          <number>500</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="tickPosition">
          <enum>QSlider::NoTicks</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="tabBeta">
     <attribute name="title">
      <string>Tab β</string>
     </attribute>
     <widget class="QWidget" name="verticalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>-1</x>
        <y>-1</y>
        <width>1891</width>
        <height>1031</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="tabBetaVerticalLayout">
       <item>
        <widget class="QPushButton" name="topButton">
         <property name="font">
          <font>
           <family>Segoe Print</family>
           <pointsize>48</pointsize>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>Top Button</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="middleButton">
         <property name="font">
          <font>
           <family>Segoe Print</family>
           <pointsize>48</pointsize>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>Middle Button</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="bottomButton">
         <property name="font">
          <font>
           <family>Segoe Print</family>
           <pointsize>48</pointsize>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>Bottom Button</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
